### ISSUE TITLE
Update the Settlement Map 

### ISSUE THREAD

**Issue by seabilwe** [kartoza/WBR-SEMP#114]
Ref: https://github.com/kartoza/WBR-SEMP/issues/51

- [x] Clean up Figure 16
- [x] Can we retain basic “background format” agreed in Fig 13 & again Fig 15, & then superimpose town & village names. The NE section has high density of villages – maybe a separate map?
- [x] Also another map Of human density gradation???
- [x] Acquire settlement data

![image](https://user-images.githubusercontent.com/56301949/216919882-881dcc63-afd3-4de0-aa8a-7e32e1fb86d0.png)

**Comment by seabilwe** [comment]
**Data Source:** http://www.sasdi.net/metaview.aspx?uuid=1f9572f479060bd3b956604e1ad52ed2
The dataset contains mesozones with the South African functional settlement typology. 
**Data Providers:** The typology dataset was developed by the CSIR as part of the National Spatial Trends Overview project (2008-2009) commissioned by the South African Cities Network.

Updated Settlement Map: 
![Settlement and Population](https://user-images.githubusercontent.com/56301949/220472733-ab012fd2-d4c1-49aa-a18b-05f22c2613e5.png)

**Comment by seabilwe** [comment]
Could not acquire clean human density graduation datasets. Most of them are raster datasets that are broken. The CSIR dataset, acquired for the settlement map, contains a field with census 2011 population information. Will suffice?
![Settlement and Population_2](https://user-images.githubusercontent.com/56301949/220472860-3393e10c-f459-40b0-a6db-30213caa15af.png)

**Comment by RAP-ZAUK** [comment]
Looks good to me, altho I wonder about the automated bins _ I would be more tempted to use
0
1 - 500
501 - 1000
1001 - 2000
2001 - 5000
5001 - 10000 etc ...

**Comment by gubuntu** [comment]
Yes definitely use cleaner bins, even better would be five quantile categories, with nice rounded boundaries. 

And for typology a colour gradient is not correct.

**Comment by seabilwe** [comment]
Updated maps:

Settlement Type
![Settlement and Population](https://user-images.githubusercontent.com/56301949/221126636-2bb71b3c-d201-49b6-8985-96944b2d9464.png)

Population
![Settlement and Population_2](https://user-images.githubusercontent.com/56301949/221126686-963be69e-ead1-4d1c-9641-a644353642ff.png)

**Comment by RAP-ZAUK** [comment]
In both the above maps we have now lost the Proposed WBR boundaries - can you please put in the polygon boundary for proposed WBR in each map. Thanks. I am also not convinced by the bin cut-offs, but it shows the pattern, so lets leave for now

**Comment by seabilwe** [comment]
**Settlement Type**
![settlement-and-population](https://user-images.githubusercontent.com/56301949/223430658-81623251-639b-42e6-86fe-3775de3ff197.png)

**Population** 
![settlement-and-population_2](https://user-images.githubusercontent.com/56301949/223430823-7d38c0ed-6d7c-4e0a-9b29-58a64c9a4945.png)


I am not quite sure on how the boundary zones fit with the two maps, there is a bit of a clutter

**Comment by seabilwe** [comment]
Updated settlement type map:
![settlement-type](https://user-images.githubusercontent.com/56301949/228181296-f9ef2044-76dc-4478-b327-edf0d7b71d88.png)

**Comment by seabilwe** [comment]
Updated population map:
![population](https://user-images.githubusercontent.com/56301949/228181439-a6a29723-5c98-4014-a3d0-f296c612081d.png)
